### PR TITLE
Adds dizziness

### PR DESCRIPTION
**Pull request by Ivniinvi**
## About The Pull Request
You are now confused for a very short period after using *spin and *flip. This stacks on multiple uses of the emote.

## Why It's Good For The Game
The effect is practically nonexistent on single, isolated uses of the emote. However, when someone spams *flip fast enough to look like they got caught in a blender, the dizziness will add up, much like it would in real life.

## Changelog
:cl:
add: The *spin and *flip emotes now cause a small amount of confusion.
/:cl:

### CODE REVIEW

**Comment by Kapu1178** [comment]
lol

**Comment by Pirill** [comment]
1984

**Comment by Ivniinvi** [comment]
Shamelessly stolen (with numeric changes)
image


It's all Ruko's fault!

**Comment by zeskorion** [comment]
Based

**Comment by Archanial** [comment]
The changelog is inacurate

**Comment by super12pl** [comment]
The fall
The fall of beestation 😔

**Comment by BrumalLake** [comment]
sounds like a tg thing since they spin clap and flip all the time when there's an antagonist near them or they are one

**Comment by OceanFish1** [comment]
Damn i wanted to add limb detachment on flip spam

**Comment by St0rmC4st3r** [comment]
Make an exception for the clown and mime

File: code/modules/mob/emote.dm [hunk 1]
```diff
@@ -44,12 +47,14 @@
...
+			L.confused += 2
 		if(iscyborg(user) && user.has_buckled_mobs())
 			var/mob/living/silicon/robot/R = user
 			var/datum/component/riding/riding_datum = R.GetComponent(/datum/component/riding)
 			if(riding_datum)
 				for(var/mob/M in R.buckled_mobs)
 					riding_datum.force_dismount(M)
 			else
-				R.unbuckle_all_mobs()
+				R.unbuckle_all_mobs()
```
Comment (Urumasi) [inline code/modules/mob/emote.dm:60]: EOL my beloved

**Pull request merged**